elife/masterless.sls

```diff
@@ -1,8 +1,64 @@
-# when running salt masterless, the minion doesn't need to run as a daemon
+#
+# only included for masterless instances
+#
+
+# when running salt masterless, minion doesn't need to run as a daemon
 disable-salt-minion:
     service.dead:
         - name: salt-minion
         - enable: false
-        - onlyif:
-            # running within vagrant
-            - test -d /vagrant
+
+# deny access to all except those in allow.all and allow.master-server
+# necessary to ensure those with access don't inadvertently grant access to others
+
+# zero out all allowed keys
+deny_all_access:
+    cmd.run:
+        - name: |
+            echo > /home/elife/.ssh/authorized_keys
+            # bootstrap user is needed to login
+            # masterless environment disables adding any other keys to bootstrap user
+            #echo > /home/ubuntu/.ssh/authorized_keys
```

Comment:
why is the second `echo` commented out?

Reply:
figured out from https://github.com/elifesciences/builder-private/pull/34/files but I would remove the `echo` at all then